UACloudLibraryServer/Controllers/ApprovalController.cs

```diff
@@ -38,7 +38,7 @@ namespace Opc.Ua.Cloud.Library
     using Microsoft.Extensions.Logging;
     using Swashbuckle.AspNetCore.Annotations;
 
-    [Authorize(AuthenticationSchemes = "BasicAuthentication")]
+    [Authorize]
```

Comment:
I was hoping that we can keep just basic auth for the API, as interactive login for e.g. the OPC Foundation OAuth2 flow won't work with the API anyway. Or make it Authorize(AuthenticationSchemes = "BasicAuthentication,AzureAD")? Then you don't need the #if !NO_AZURE_AD, which is a double-negative anyway! :-)

Reply:
Without this, swagger and graphiql UIs don't work.

Reply:
Got it to work: will leave as it was.

Reply:
Done.